src/examples/table/TableScroll.tsx

```diff
@@ -7,14 +7,19 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import getScrollbarSize from 'dom-helpers/scrollbarSize';
+import { getColor } from '@zendeskgarden/react-theming';
 import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
 
-const SCROLLBAR_SIZE = getScrollbarSize();
+const StyledHeaderRow = styled(HeaderRow)`
+  border-bottom: none;
+`;
 
-const StyledSpacerCell = styled(HeaderCell)`
-  padding: 0;
-  width: ${SCROLLBAR_SIZE}px;
+const StyledHead = styled(Head)`
+  box-sizing: border-box;
+  position: sticky;
+  top: 0;
+  box-shadow: inset 0 -1px 0 ${props => getColor('neutralHue', 300, props.theme)};
```

Comment:
```suggestion
  box-shadow: inset 0 -${props => props.theme.borderWidths.sm} 0 ${props => getColor('neutralHue', 300, props.theme)};
```

per https://github.com/zendeskgarden/website/pull/452#discussion_r1038360871